go.mod

```diff
@@ -4,19 +4,17 @@ go 1.16
 
 require (
 	github.com/imdario/mergo v0.3.10 // indirect
-	github.com/kubernetes-csi/csi-lib-utils v0.9.1
-	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.0.0
+	github.com/kubernetes-csi/csi-lib-utils v0.11.0
+	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.0.0
```

Comment:
Let's wait for v6.0.1. There's an issue we need to fix there.

Reply:
its updated to v6.0.1